lib/Dropdown/DropdownContent.js

```diff
@@ -4,32 +4,76 @@ import PropTypes from 'prop-types';
 import { GATHER_UI_DROPDOWN } from './consts';
 
 class DropdownContent extends Component {
+  getTopPosition = (pos, bounds, contentBounds) => {
```

Comment:
This is the dropdown auto position changes. The process goes as follows;

- get what the top and left positions will be and run them through `getTopPosition` and `getLeftPosition`
- if they exceed then alter the position
- if the altered position goes off-screen the other way then alter the position
- if the position is always good, just remain as default